### ISSUE TITLE
Is there a way play acoustic-ish sounds with webdx7

### ISSUE THREAD

**Issue by christopher-millon-abide** [webaudiomodules/webdx7#6]

**Comment by jariseon** [comment]
it kind of depends. FM synthesis can mimic the dynamic spectral aspects of acoustic instruments, but it is still an abstract synthesis method. if you need more accurate reproduction, sample playback might be what you are after. i'm releasing `fluidsynth` WAM soon. will update here once that is available.

**Comment by christopher-millon-abide** [comment]
Thank you, can't wait to try it out!

**Comment by ChristopherMillon** [comment]
Would it support the sustain pedal?

**Comment by jariseon** [comment]
yes it would. fluid's midi implementation looks pretty nice: https://github.com/FluidSynth/fluidsynth/wiki/FluidFeatures
